src/controller/java/CHIPDeviceController-JNI.cpp

```diff
@@ -108,27 +85,17 @@ jclass sChipDeviceControllerExceptionCls = NULL;
 chip::NodeId kLocalDeviceId  = chip::kTestControllerNodeId;
 chip::NodeId kRemoteDeviceId = chip::kTestDeviceNodeId;
 
-#if CONFIG_NETWORK_LAYER_BLE
-
-chip::Ble::BleLayer * GetJNIBleLayer()
-{
-    return &sBleLayer;
-}
-
-#endif
-
```

Comment:
@austinh0 Can this be removed?

Reply:
Along with other methods referencing BLE layer

Reply:
This method is called at src/controller/java/AndroidDeviceControllerWrapper.cpp, and not all related code are moved to platform/android, so no need anymore.

    initParams.inetLayer                      = inetLayer;
    // move bleLayer into platform/android to share with app server
    initParams.bleLayer = DeviceLayer::ConnectivityMgr().GetBleLayer();

    wrapper->InitializeOperationalCredentialsIssuer();